src/Services/OrderService.php

```diff
@@ -388,10 +388,13 @@ public function getOrderOverviewListForMyAccount(int $page = 1, int $items = 50)
                     {
                         $creationDate = $creationDateData->date;
                     }
-    
+
+                    $highlightNetPrices = $orderTotalsService->highlightNetPrices($order);
+                    $displayTotal = $highlightNetPrices ? $totals['totalNet'] : $totals['totalGross'];
```

Comment:
Wenn du schon die kurze Schreibweise nimmst, kannst du die Abfrage auch gleich unten bei 'total' => ... reinschreiben anstatt es vorher noch in ne Variable zu schreiben.

Reply:
done